From 13297949e826637e3bd36bee2e89e83544b4e72f Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Sat, 31 May 2014 19:31:42 +0000
Subject: [PATCH] Remove Qt3D dependency

---
 src/src.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/src.pro b/src/src.pro
index 26d111d..b17134f 100644
--- a/src/src.pro
+++ b/src/src.pro
@@ -11,7 +11,7 @@ SUBDIRS += plugins
 #SUBDIRS += positioning_doc_snippets
 
 #no point in building QtLocation without Qt3D
-qtHaveModule(3d) {
+qtHaveModule(quick) {
     SUBDIRS += 3rdparty
 
     location.depends = positioning 3rdparty
-- 
1.8.3-rc3

